index.bs

```diff
@@ -144,6 +128,17 @@ partial interface DocumentOrShadowRoot {
 	</dd>
 </dl>
```

Comment:
The dl here has a statement of fact, not a normative requirement. It's also at the wrong level, talking about the attribute instead of the "adopted stylesheets" concept. I would suggest deleting it, and instead putting the `<dfn>` around the getter.